src/types/actions.js

```diff
@@ -35,6 +35,7 @@ export type DataSource =
   | 'none'
   | 'from-file'
   | 'from-addon'
+  | 'unpublished' // This is an alias for 'from-addon' until we phase that one out.
```

Comment:
Suggestion: This comment is nice, but it could be even better with a bit more context on why it was added. It would be good to document `from-addon` as well in this patch, as it's adding more complexity to the DataSource situation.

Follow-up: We really should document each of these data sources here, and rename `DataSource` to something more correct, like `PageViewSlug`.